### DISCUSSION TITLE
Page buttons for Kits by default

### DISCUSSION THREAD

**Discussion by CloudeLecaw** [Zrips/CMI#6924] [Enhancement]
I dont know how often this happen but if you have the inventory filled with kits and you wanne show more
is it possible to add the Pagebutons like we know them from the Homes and Warp GUI?

![image](https://user-images.githubusercontent.com/12080017/169963304-a4dd6bea-3a5a-45fa-b56d-1f0665d2175e.png)
 
A user in CMI was complain about that he cant showall his kis to his players
Well here is his screenshot

![image](https://user-images.githubusercontent.com/12080017/169963484-a9c17e31-e8fe-41c5-889c-5cf3c05e8580.png)

So it looks like the buttons are not added by default.
well i think you still then can add them manually if needed... right?

**Comment by mrfloris** [comment]
I did a test for this, they won't show by default when there's not enough to justify a page 2.
If there's more than fit on page 1, the buttons show up. 

I think that's a reasonable default behavior.
